packages/hooks/dao/useDAOProposals.ts

```diff
@@ -51,7 +51,7 @@ export const useDAOProposals = (daoId: string | undefined) => {
   const { daoProposals: cosmWasmDAOProposals, ...cosmWasmOther } =
     useCosmWasmDAOProposals(daoId);
   const { data: gnoDAOProposals, ...gnoOther } = useQuery(
-    [daoProposalsQueryKey(daoId), NetworkKind.Gno],
+    [...daoProposalsQueryKey(daoId), NetworkKind.Gno],
```

Comment:
```suggestion
    daoProposalsQueryKey(daoId),
```
the network kind is already encoded in the `daoId` since the network prefix is unique to the network and the kind can be derived from network info

Reply:
done